test/unit/core.js

```diff
@@ -1522,7 +1522,37 @@ testIframe(
 	}
 );
 
-QUnit[ includesModule( "deferred" ) ? "test" : "skip" ]( "jQuery.readyException (original)", function( assert ) {
+QUnit[ includesModule( "deferred" ) && window.queueMicrotask ? "test" : "skip" ]( "jQuery.readyException (microtask)", function( assert ) {
+	assert.expect( 2 );
+
+	var done = assert.async();
+	var expected = "Error in jQuery ready #" + Math.random().toString( 36 ).slice( -7 );
+
+	this.sandbox.stub( jQuery, "readyException" ).callsFake( function( error ) {
+		assert.strictEqual( error.message, expected, "jQuery.readyException called with unexpected error" );
+		window.setTimeout( function() {
+			throw error;
+		} );
+	} );
```

Comment:
Why mocking `jQuery.readyException`? Can't this test be done in a similar way as the `setTimeout` one? In that case, maybe we could just keep a single test?